README.md

```diff
@@ -31,28 +31,32 @@
 - [June, 2023] [**Two new clouds supported**](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html): Samsung SCP and Oracle OCI!
 - [April, 2023] **[**SkyPilot YAMLs released**](./llm/vicuna/) for finetuning & serving the Vicuna model with a single command**!
 - [March, 2023] **[Vicuna LLM chatbot](https://lmsys.org/blog/2023-03-30-vicuna/) trained** [**using SkyPilot**](./llm/vicuna/) **for $300 on spot instances!** 
-- [March, 2023] *Serve* your own LLaMA LLM chatbot (not finetuned) on any cloud: [**example**](./llm/llama-chatbots/), [**repo**](https://github.com/skypilot-org/sky-llama)
+- [March, 2023] Serve your own LLaMA LLM chatbot (not finetuned) on any cloud: [**example**](./llm/llama-chatbots/), [**repo**](https://github.com/skypilot-org/sky-llama)
 ----
 
-SkyPilot is a framework for easily and cost effectively running ML workloads[^1] on any cloud. 
+SkyPilot is a framework for running LLMs, AI, and Batch jobs on any cloud, offering maximum cost savings, highest GPU availability, and managed execution.
```

Comment:
```suggestion
SkyPilot is a framework for running LLMs, AI, and batch jobs on any cloud, offering maximum cost savings, highest GPU availability, and managed execution.
```

Seems like batch job is not a proper noun - [Google results](https://www.google.com/search?q=Batch+jobs&oq=Batch+jobs&aqs=chrome..69i57j69i59l2j69i61l2.1031j0j1&sourceid=chrome&ie=UTF-8#ip=1)